### ISSUE TITLE
Exception due to null contentDocument

### ISSUE THREAD

**Issue by eyalroz** [eyalroz/bidimailui#52]
In bidimailui-composer.-js, we have:
```
setDocumentDirection : function(direction) {

    let contentFrame = document.getElementById("content-frame");
    if (contentFrame) {
      document.getElementById("content-frame").contentDocument.documentElement.style.direction = direction;
      document.getElementById("content-frame").contentDocument.body.style.direction = direction;
    }
    if (BiDiMailUI.App.versionIsAtLeast("71")) {
      document.getElementById("msgSubject").style.direction = direction;
    }
    else {
      // We can't use the dir attribute of the subject textbox / html:input, like we
      // do for the message body, since XUL elements' dir attribute means something
      // else than this attribute for HTML elements. But we can set it for its input
      // field...
      document.getElementById("msgSubject").inputField.style.direction = direction;
    }
    BiDiMailUI.Prefs.set("compose.last_used_direction", direction);
  },
```
and this assumed that if the contentFrame is not null, the contentDocument must also be non-null. Sadly, and strangely, that is not the case... :-(

Need to account for this situation.

**Comment by eyalroz** [comment]
Seeing a null content document sometimes when creating a new empty message, and then not seeing it if I create a second message.

**Comment by eyalroz** [comment]
Seems to have been resolved by one of John Bieling's PRs: #58 and #60 , which accommodate the ID change in TB 100 from  "content-frame" to "messageEditor".
